### PR TITLE
Synchronize the github.io site for submission

### PR DESCRIPTION
**Pull request by mrshirts**

### CODE REVIEW

File: _author_instructions/01_policies.md [hunk 1]
```diff
@@ -166,15 +166,14 @@ Posting to a preprint server can be done at any time prior to submission.
...
-This should be uploaded as a "Supporting File", with the descriptor "Presubmission Letter"
-  * A representative image as noted above which will appear on the article's page on the journal.
-  * Any Supporting Files that are intended for distribution with the final paper should also be uploaded at this time. There is no explicit limit to the number of files that can be added, though if there are many similar data files, they should be uploaded as a `.tgz`ed or `.zip`ed directory and include a README.md or README.txt file.
+  * As part of the submission form, authors will suggest 4-5 reviewers.
+  * As a single PDF file, a copy of their presubmission letter, appended with the response recieved from the Section Lead Editor approving the presubmission and specifying any changes or modifications to the scope the editors requested. The authors should also include a description of any deviations, if any, from what was laid out in the presubmission letter, with a rationale for why the changes. This should be uploaded as a "Supporting File", with the descriptor "Presubmission Letter".
```
Comment (dmzuckerman) [inline _author_instructions/01_policies.md]: Some redundancy and a missing phrase in the sentence, "The authors should also include a description of any deviations [, if any,] from what was laid out in the presubmission letter, with a rationale for why the changes WERE MADE."  Brackets and capitalized phrase added by me.

Comment (mrshirts) [reply _author_instructions/01_policies.md]: Thanks, fixed.
